src/Menu/Menu.js

```diff
@@ -242,7 +242,7 @@ class Menu extends Component {
 
     const filteredChildren = this.getFilteredChildren(this.props.children);
     const focusedItem = filteredChildren[focusIndex];
-    if (focusedItem.props.menuItems && focusedItem.props.menuItems.length > 0) {
+    if (!!focusedItem && focusedItem.props.menuItems && focusedItem.props.menuItems.length > 0) {
```

Comment:
Is the following check enough? `if (focusedItem && `

Reply:
suppose i don't see why not. I sometimes go overboard with ensuring i've got a bool :)

Reply:
btw, the minified file translate it to o&&o.props.menuItems&&o.props.menuItems.length>0
so `!!` really does not matter :)